app/utils/Config.scala

```diff
@@ -0,0 +1,43 @@
+package utils
+
+import org.yaml.snakeyaml.Yaml
+import scala.io.Source
+import scala.collection.JavaConversions._
+import scala.collection.mutable.Map
+import java.util.LinkedHashMap
+
+class Config(config_fn: String) {
+  private var api_key = ""
```

Comment:
Do you think it would be a better idea to make this a `val` and just hardcode the config filename? I don't like that this can be changed.